beta/src/pages/learn/conditional-rendering.md

```diff
@@ -100,13 +100,13 @@ export default function PackingList() {
 
 </Sandpack>
 
-Try editing what gets returned in either case, and see how the result changes!
+किसी भी मामले में जो वापस मिलता है उसे संपादित करने का प्रयास करें, और देखें कि परिणाम कैसे बदलता है!
```

Comment:
`संपादित करने` => `बदलने`

let me know your thoughts on this? Which one would make more sense here. I am not very sure about this.

Reply:
`संपादित करने` is complex hindi where as `बदलने` is commonly used.